eclass/flag-o-matic.eclass

```diff
@@ -533,33 +585,51 @@ test-flag-PROG() {
 # @USAGE: <flag>
 # @DESCRIPTION:
 # Returns shell true if <flag> is supported by the C compiler, else returns shell false.
-test-flag-CC() { test-flag-PROG "CC" c "$@"; }
+test-flag-CC() { _test-flag-PROG "CC" c "$@"; }
```

Comment:
This is very minor, but is there any reason for the inconsistent quoting? Either `CC` shouldn't be quoted (which I'd slightly prefer), or `c` should.

Same for the other functions below.